### DISCUSSION TITLE
女仆行为的一些建议

### DISCUSSION THREAD

**Discussion by Eldoradocsk** [TartaricAcid/TouhouLittleMaid#344] [Suggestions & Ideas]
第一个，希望能够实现不同的女仆使用不同的语音包
第二个，希望能够单独设置女仆的声音间隔
第三个，也是我发这次反馈的主要原因：女仆使用弓箭时如果周围有岩浆池，为了保证攻击在接近怪物或者风筝的时候无视伤害直接跳入岩浆中，导致我多次为了救人跳进岩浆用魂符（开了死亡不掉落）

**Comment by Eldoradocsk** [comment]
然后还有一个，希望能做个归家道具，夜晚自动回到特定的地方，比如床附近，类似于罗盘，只不过没有罗盘的范围限制

**Comment by TartaricAcid** [comment]
1. 比较困难，原版声音系统需要重写
2. 可行，但是改起来比较麻烦
3. 额，AI 方面确实比较头疼，我尽力解决吧
